packages/webapp/components/layouts/PlusLayout/PlusHeader.tsx

```diff
@@ -9,10 +9,28 @@ import { useRouter } from 'next/router';
 import { webappUrl } from '@dailydotdev/shared/src/lib/constants';
 import { LogoWithPlus } from '@dailydotdev/shared/src/components/Logo';
 import { useViewSize, ViewSize } from '@dailydotdev/shared/src/hooks';
+import {
+  Typography,
+  TypographyType,
+} from '@dailydotdev/shared/src/components/typography/Typography';
+import { usePaymentContext } from '@dailydotdev/shared/src/contexts/PaymentContext';
+
+export const PlusFreeTrialAlert = (): ReactElement => {
+  return (
+    <div className="min-w-full bg-status-success px-6 py-4 text-center text-black">
```

Comment:
I just saw this exact component up one level, why is it here again?

Reply:
Forgot to delete this, doing it now 🙏